draft-ietf-httpbis-encryption-encoding.md

```diff
@@ -366,13 +366,16 @@ avoided).
 # Crypto-Key Header Field {#crypto-key}
 
 A Crypto-Key header field can be used to describe the input keying material
-used in the Encryption header field.
+used in the Encryption header field. Crypto-Key is a composite field,
+where multiple protocols may store information. Each parameter set
+shall be separated by a comma and should be identified by a "keyid"
+parameter.
 
 The Crypto-Key header field uses the extended ABNF syntax defined in Section 1.2
 of [RFC7230] and the `parameter` and `OWS` rules from [RFC7231].
 
 ~~~ abnf7230
-  Crypto-Key = #crypto_key_params
+  Crypto-Key = crypto_key_params *(COMMA crypto_key_params)
```

Comment:
This change isn't correct, or even needed.  The `#` has the same effect as what you are doing here, but it also allows for whitespace in the usual ways.